MVP/MVPUI/LiveIdLogOn.xaml

```diff
@@ -1,9 +1,13 @@
-﻿<?xml version="1.0" encoding="utf-8" ?>
-<ContentPage xmlns="http://xamarin.com/schemas/2014/forms"
+<?xml version="1.0" encoding="utf-8" ?>
+<local:ModalPage Title="Sign in" xmlns="http://xamarin.com/schemas/2014/forms"
```

Comment:
This was done for iOS because it behaves badly when modal pages and status/navigation bar colors are concerned. What would happen is when you tapped the Sign in button (white status bar) the modal page shows (black status bar) but going back the statusbar won't turn white again. This is fixed through a custom renderer based on ModalPage. If there's a way to check if ContentPage is currently shown modal I'd love to hear it.

Reply:
So this wont be a problem if we wrap the page in a NavigationPage! I added this to the guidelines. This will ensure that it is correct.

Reply:
Shouldn't "Close"/"Cancel" go in the top left for iOS? My understanding is top right is for things like "Save"/"Done" or to take some type of action.

Reply:
Most likely should. Will change it ASAP.

Reply:
Yeah, so I think we should not add a custom renderer for that as it isn't really needed and will cause some weird stuff on other platforms.

On the bottom of a page can be Save button and then on the top right will be Close/Cancel always. This just simplifies it across all platforms